cmd/mcp-grafana/main.go

```diff
@@ -34,7 +34,7 @@ type disabledTools struct {
 
 	search, datasource, incident,
 	prometheus, loki, alerting,
-	dashboard, oncall, asserts, sift bool
+	dashboard, oncall, asserts, sift, teams bool
```

Comment:
```suggestion
	dashboard, oncall, asserts, sift, admin bool
```

IMO we should start with this being a larger 'admin' category for things like teams, permissions, roles. We can make it more granular later but 'teams' is quite vague.

Reply:
renamed to admin. it just makes sense